src/BcPrebidVast.js

```diff
@@ -65,6 +66,16 @@ function doPrebid(options, callback) {
 	}
 }
 
+function specifyBidderAliases(bidderAliases, bc_pbjs) {
```

Comment:
Please add a comment to summarize what the arguments are and what they mean.  Also specify in the comment what the function will do